src/sentry/static/sentry/app/views/projectInstall/createProject.jsx

```diff
@@ -241,11 +231,9 @@ class CreateProject extends React.Component {
         ruleId = ruleData.id;
       }
       this.trackIssueAlertOptionSelectedEvent(
-        organization,
         projectData,
         defaultRules,
         shouldCreateCustomRule,
-        platform,
```

Comment:
This was wrong. This parameter was not part of the `trackIssueAlertOptionSelectedEvent` signature.